src/components/MedicationGroup.js

```diff
@@ -52,6 +58,19 @@ export default function MedicationGroup({
             <img src={editIcon} alt="bearbeiten" height="16px" />
           </IconButton>
         </>
+      ) : (
+        <IconButton
+          right="10px"
+          bottom="10px"
+          onClick={() => handleCheckClick(id)}
+          position="absolute"
+        >
+          {check ? (
+            <img src={checkIcon} alt="löschen" height="20px" />
+          ) : (
+            <img src={checkIconDisabled} alt="löschen" height="20px" />
```

Comment:
Both alt texts are 'löschen'. I believe it would be more accessible if better described:)

Reply:
yes, this is mistake I made